From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oskar Skog <oskar@oskog97.com>
Date: Mon, 25 Sep 2023 19:21:14 +0300
Subject: [PATCH] Set CCSHARED=-fPIC in configure

---
 configure    | 1 +
 configure.ac | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configure b/configure
index db10d497b4719543be8ffc65b16f3931c229dad9..cae3c9045b11d39f803625addfcc556201f26a1c 100755
--- a/configure
+++ b/configure
@@ -11044,6 +11044,7 @@ then
 fi;;
 	FreeBSD*|NetBSD*|OpenBSD*|DragonFly*) CCSHARED="-fPIC";;
 	Haiku*) CCSHARED="-fPIC";;
+	SerenityOS*) CCSHARED="-fPIC";;
 	OpenUNIX*|UnixWare*)
 		if test "$GCC" = "yes"
 		then CCSHARED="-fPIC"
diff --git a/configure.ac b/configure.ac
index 61cbb6d0f3b31192495d58176c88c2f4c9700cd0..a36c62bab8f3ea3e6568bbeb5e45893c44c6a22f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3270,6 +3270,7 @@ then
 		]);;
 	FreeBSD*|NetBSD*|OpenBSD*|DragonFly*) CCSHARED="-fPIC";;
 	Haiku*) CCSHARED="-fPIC";;
+	SerenityOS*) CCSHARED="-fPIC";;
 	OpenUNIX*|UnixWare*)
 		if test "$GCC" = "yes"
 		then CCSHARED="-fPIC"
